core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/util/CamelMavenUtils.java

```diff
@@ -212,6 +212,16 @@ public String getWildFlyCamelVersionFromMaven(IProject project) {
 		}
 		return null;
 	}
+
+	/**
+	 * tests if a given version contains the Red Hat brand string
+	 * 
+	 * @param version
+	 * @return
+	 */
+	public boolean isRedHatBrandedVersion(String version) {
+		return version.toLowerCase().indexOf(".redhat-") != -1;
```

Comment:
there was a time, ".fuse-" suffix was also used.
Is it still the case?

Reply:
I don't think we support that any more nor should it be used any more.